### PR TITLE
Fixes #2660 remove estimated ga and use PMA instead

### LINKED ISSUE #2660: Remove Estimated GA and use PMA instead
FYI @Yuri05 @PavelBal as discussed today

### PR DESCRIPTION
**Pull request by msevestre**

### CODE REVIEW

File: src/Db/Diff/PKSimDB_diff.txt [hunk 1]
```diff
@@ -2,7 +2,8 @@ DELETE FROM tab_calculation_method_rates WHERE calculation_method='BMI' AND form
 INSERT INTO tab_calculation_method_rates(calculation_method,formula_rate,formula,dimension) VALUES('DiseaseStates','PARAM_ChildPughScore','0','Dimensionless');
 INSERT INTO tab_calculation_method_rates(calculation_method,formula_rate,formula,dimension) VALUES('ExpressionParameters','TableFormulaWithXArgument_OntogenyFactor',NULL,'Dimensionless');
 INSERT INTO tab_calculation_method_rates(calculation_method,formula_rate,formula,dimension) VALUES('ExpressionParameters','TableFormulaWithXArgument_OntogenyFactorGI',NULL,'Dimensionless');
-UPDATE tab_calculation_method_rates SET formula='Age * 365.25/7 + GA' WHERE calculation_method='GFR_Calculation' AND formula_rate='PARAM_PostMenstrualAge';
+UPDATE tab_calculation_method_rates SET formula='((PMA*7/365.25)^Hill/(TM50^Hill+(PMA*7/365.25)^Hill)*(1-GFR_premat)+GFR_premat) * GFR_mat / V_kid_std * Renal_aging_scaling_factor' WHERE calculation_method='GFR_Calculation' AND formula_rate='PARAM_GFR_Spec';
```
Comment (msevestre) [inline src/Db/Diff/PKSimDB_diff.txt:5]: @Yuri05 This was the only place where this was used. The PMA in the formula is assumed to be in weeks. Well it's in year and we move it to weeks by "hand"

**Review by msevestre — COMMENTED** [review]
simple easy. I even think we do not need any conversion but I am not sure? Thoughts?

File: src/Db/Dump/PKSimDB_dump.txt [hunk 2]
```diff
@@ -3663,8 +3663,7 @@ INSERT INTO tab_calculation_method_rates VALUES('FormulationParticles','PARAM_Pa
 INSERT INTO tab_calculation_method_rates VALUES('FormulationParticles','PARAM_ParticleSizeDistribution','0','Dimensionless');
 INSERT INTO tab_calculation_method_rates VALUES('FundamentalConstants','PARAM_MW_Water','1.801528e-008','Molecular weight');
 INSERT INTO tab_calculation_method_rates VALUES('GFR_Calculation','PARAM_GFR','GFRspec * Weight_Kidney','Flow');
-INSERT INTO tab_calculation_method_rates VALUES('GFR_Calculation','PARAM_GFR_Spec','(PMA^Hill/(TM50^Hill+PMA^Hill)*(1-GFR_premat)+GFR_premat) * GFR_mat / V_kid_std * Renal_aging_scaling_factor','Flow per weight organ');
-INSERT INTO tab_calculation_method_rates VALUES('GFR_Calculation','PARAM_PostMenstrualAge','Age * 365.25/7 + GA','Age in weeks');
+INSERT INTO tab_calculation_method_rates VALUES('GFR_Calculation','PARAM_GFR_Spec','((PMA*7/365.25)^Hill/(TM50^Hill+(PMA*7/365.25)^Hill)*(1-GFR_premat)+GFR_premat) * GFR_mat / V_kid_std * Renal_aging_scaling_factor','Flow per weight organ');
```
Comment (Yuri05) [inline src/Db/Dump/PKSimDB_dump.txt:3666]: it should be `PMA * 365.25 / 7`, not `PMA*7/365.25`

File: src/Db/Diff/PKSimDB_diff.txt (hunk 1 above)
Comment (Yuri05) [reply src/Db/Diff/PKSimDB_diff.txt:5]: it should be `PMA * 365.25 / 7`, not `PMA*7/365.25`

Comment (msevestre) [reply src/Db/Diff/PKSimDB_diff.txt:5]: Indeed

**Comment by msevestre** [comment]
done

**Comment by Yuri05** [comment]
> simple easy. I even think we do not need any conversion but I am not sure? Thoughts?

Yes, GFR_Spec, PMA, estimated gestational Age are all in the individual, so I think too we don't need any conversion.

**Review by Yuri05 — APPROVED** [review]

**Pull request merged**